src/components/style.js

```diff
@@ -51,7 +51,15 @@ const Style = React.createClass({
         const completeSelector = scopeSelector
           ? selector
             .split(',')
-            .map(part => scopeSelector + ' ' + part.trim())
+            .map(part => {
+            	part = part.trim();
+            	if (part.startsWith('&')) {
```

Comment:
Radium supports the same browser matrix as React, so `startsWith` will need to be replaced with something more broadly supported like `part.substr(0, 1) === '&'`